dts/bindings/flash_controller/nuvoton,npcx-fiu-nor.yaml

```diff
@@ -28,9 +28,15 @@ properties:
     type: int
     required: true
     description: Mapped memory address of direct read access for spi nor flash.
-  min-erase-size:
+  max-erase-size:
     type: int
     default: 0x10000
+    description: Maximum erase size of spi nor flash.
+    enum:
+      - 0x10000 # 64KB (Block Erase)
```

Comment:
Add ` - 0x1000  # 4KB (Sector Erase)`? Or we needn't `BUILD_ASSERT` checks at line 613.

Reply:
Yes but if size of memory needs to be erased is more than 0x10000, it's optimal to erase it at a time right, than 10 * 0x1000 block erase.

Reply:
@MulinChao does it even make sense to keep this and `min-size` configurable or is it just a leftover from the spi-nor implementation?

Reply:
@fabiobaltieri  At the beginning, the purpose of `min-erase-size` is to let users select the minimum size of flash erasing for better performance simply. But yes, we might encounter users who want to update a specific field and the whole ec firmware image in the same application. Hence, I think removing `min-erase-size` property and implementing this like [soc_flash_b91.c](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/flash/soc_flash_b91.c#L90) or [spi_nor.c](erase_types) is a good idea. Actually, this PR is similar to spi_nor.c approach if [CONFIG_SPI_NOR_SFDP_MINIMAL](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/flash/spi_nor.c#L154) is enabled.

Reply:
Brilliant, thanks @MulinChao, yeah I was trying to dig into the history of `SPI_NOR_FLASH_LAYOUT_PAGE_SIZE`, which I guess is where this originates from, and it seems to be related to `CONFIG_FLASH_PAGE_LAYOUT`, but then the `layout` field just uses `SPI_NOR_PAGE_SIZE` unconditionally so it feels like this configuration is a leftover.

Reply:
> So we can drop both the min and max property, right?

Yes, I think so.

Reply:
Removed min and max erase size, thank you